angular/_margins.scss

```diff
@@ -36,5 +37,142 @@
   }
 
   @mixin bodyfont(){
-    font-family: "Open Sans", Arial, Helvetica, sans-serif;
-  }
\ No newline at end of file
+    font-family: "Open Sans", Arial, Helvetica, sans-serif; 
+  }
+  @mixin darktheme(){
+/* Side navigation*/
+    .mat-drawer{
+        background-color: black; 
```

Comment:
What does this rule do? The background of the drawer is A400 as defined in the rule below. Also, do not use 'black'...use pxb-black a700